src/bin/casr-cluster.rs

```diff
@@ -416,10 +416,69 @@ fn dedup(indir: &Path, outdir: Option<PathBuf>) -> error::Result<(usize, usize)>
     Ok((before, after))
 }
 
+/// Merge sub directory to main
+///
+/// # Arguments
+///
+/// * `sdir` - path to sub directory
+///
+/// * `mdir` - path to main directory
```

Comment:
Maybe, better naming? Merge input directory into output directory?

Reply:
LibFuzzer uses this notation [dir1 [dir2 ...] ].  Sub dir is bad, because someone could think, that main dir should contain sub dir. Maybe new/storage, dir2/dir1 input/output... Output also an input though, but maybe it is the most relevant.

Reply:
Renamed into new/stroage